packages/forklift-console-plugin/package.json

```diff
@@ -58,6 +57,6 @@
     "webpack-dev-server": "^4.7.4"
   },
   "msw": {
-    "workerDirectory": "generated"
+    "workerDirectory": "dist"
```

Comment:
@yaacov  @sjd78 
This change  looks problematic to me:
1. the file gets injected into dist folder behind the scenes without WebPack tooling. Usual flow is compilation or copy-webpack-plugin. This i.e. makes it not visible to dev server (in my scenario)
2. the files in `dist` are often removed/re-generated - we need now to remember to re-create the service worker in all such cases

Reply:
This is the documented way [1] ,IMHO it's better then creating the a file somewhere else and then copying it to dist.

The current method has some drawbacks
a. documentation suggest to copy it directly to the public (in our case ./dist) folder, it makes sense in this case to follow the instructions.
b. it creates a "generated" directory, "generated" is a general purpose name to use when you don't have other choice, this is not the case when creating the msw server, we know what it is and why it's there.

> the file gets injected into dist folder behind the scenes

no, the documented method of running the `init` script directory to the dist script is more clear and visible than adding a hidden serving path with `webpack.conf` file, IMHO the added dir there is more problematic visibility vise, for me it's much more visible to see the `init` script run while building the code in the `package.json` file then to know to search a line in the  webpack config file.

> the files in dist are often removed/re-generated - we need now to remember to re-create the service worker in all such cases

no, the file gets copied to the dist directory automatically on each dev server build, it's more visible because the copy process is called on each build, and it's more safe because it does not require changed to the webpack server.

[1] https://mswjs.io/docs/getting-started/integrate/browser#setup